evap/staff/templates/staff_questionnaire_form.html

```diff
@@ -49,9 +49,18 @@ <h5 class="card-title">{% trans 'Questions' %}</h5>
                                     {% else %}
                                         <td></td>
                                     {% endif %}
-                                    {% for field in form_element.visible_fields %}
-                                        <td>{% include 'bootstrap_form_field_widget.html' with field=field %}</td>
-                                    {% endfor %}
+                                    <td>{% include 'bootstrap_form_field_widget.html' with field=form_element.text_de %}</td>
+                                    <td>{% include 'bootstrap_form_field_widget.html' with field=form_element.text_en %}</td>
+                                    <td class="question-type">
+                                        {% include 'bootstrap_form_field_widget.html' with field=form_element.type %}
+                                        <div class="form-check mt-1">
+                                            <input class="form-check-input allow-textanswer-checkbox" id="{{ form_element.allow_textanswers.id_for_label }}" name="{{ form_element.allow_textanswers.html_name }}"
+                                                type="checkbox"{% if form_element.allow_textanswers.value %} checked{% endif %}
+                                                {% if form_element.allow_textanswers.field.disabled %} disabled{% endif %} />
+                                            <label class="form-check-label" for="{{ form_element.allow_textanswers.id_for_label }}">{{ form_element.allow_textanswers.label }}</label>
+                                        </div>
+                                    </td>
+                                    <td>{% include 'bootstrap_form_field_widget.html' with field=form_element.DELETE %}</td>
```

Comment:
Why is it necessary to unroll the loop? I see that we somehow need to reference the input, but I don't see why the full unroll is necessary for that.

Reply:
i used this to put the checkbox under the dropdown:
![image](https://user-images.githubusercontent.com/1781719/114888507-96649300-9e09-11eb-98db-25bfe76125b0.png)

otherwise, it would be placed into its own `td` and would not have a label:
![image](https://user-images.githubusercontent.com/1781719/114888600-aed4ad80-9e09-11eb-8937-868bdf4223ea.png)